regression_detector.py

```diff
@@ -158,6 +159,31 @@ def process_regressions_into_gh_issue(regressions_dict, owner: str, output_path:
         f.write(issue_body)
 
 
+def get_best_start_date(latest_metrics_jsons, end_date: str) -> Optional[datetime]:
```

Comment:
```suggestion
def get_best_start_date(latest_metrics_jsons: List[str], end_date: str) -> Optional[datetime]:
```
is this the right type annotation?

Reply:
Yes it is correct, will fix.